src/adhocracy_core/adhocracy_core/websockets/server.py

```diff
@@ -51,6 +51,13 @@ def subscribe(self, client: Hashable, resource: IResource) -> bool:
         if self.is_subscribed(client, resource):
             return False
         path = resource_path(resource)
+
+        # ensure that no private info leaks to the outside
+        if path.startswith('/principals/') or path.startswith('/catalogs/'):
+            logger.info('Unallowed subscribtion')
+            raise WebSocketError('forbidden', 'You are not allowed to '
+                                 'subscribe on {}'.format(path))
+
```

Comment:
Yves told me there are some usability problem when websocket are disabled.

I would appreciate you work on it since you have a deeper understanding of the websocket server. Or we work together on it.